tests/test_router_bgp.rb

```diff
@@ -469,66 +469,52 @@ def test_default_disable_policy_batching
   end
 
   def test_disable_policy_batching_ipv4
-    bgp = setup_default
-    if platform == :ios_xr
-      assert_nil(bgp.disable_policy_batching_ipv4)
+    if node.product_id[/ios_xr|N(5|6|7)/]
+      b = RouterBgp.new(1)
+      assert_nil(b.disable_policy_batching_ipv4)
+      assert_nil(b.default_disable_policy_batching_ipv4)
       assert_raises(Cisco::UnsupportedError) do
-        bgp.disable_policy_batching_ipv4 = 'xx'
+        b.disable_policy_batching_ipv4 = 'xx'
       end
-    else
-      bgp.disable_policy_batching_ipv4 = 'xx'
-      assert_equal('xx', bgp.disable_policy_batching_ipv4,
-                   "bgp disable_policy_batching_ipv4 should be set to 'xx'")
-      bgp.disable_policy_batching_ipv4 = \
-        bgp.default_disable_policy_batching_ipv4
-      assert_empty(bgp.disable_policy_batching_ipv4,
-                   'bgp disable_policy_batching_ipv4 should be empty')
+      return
     end
-    bgp.destroy
-  end
-
-  def test_default_disable_policy_batching_ipv4
     bgp = setup_default
-    if platform == :ios_xr
-      assert_nil(bgp.default_disable_policy_batching_ipv4,
-                 'disable_policy_batching_ipv4 default value should be nil')
-    else
-      assert_equal(bgp.default_disable_policy_batching_ipv4,
-                   bgp.disable_policy_batching_ipv4,
-                   'disable_policy_batching_ipv4 default value should be empty')
-    end
+    bgp.disable_policy_batching_ipv4 = 'xx'
+    assert_equal('xx', bgp.disable_policy_batching_ipv4,
+                 "bgp disable_policy_batching_ipv4 should be set to 'xx'")
+    bgp.disable_policy_batching_ipv4 = \
```

Comment:
Here and below for `ipv6` your are explicitly setting the default value, but we should also have a test for the default value right after we create the new object.